src/components/searchWrapper/searchForm/chipItem/index.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'substance';
+
+class ChipItem extends Component {
+  /**
+   * Render the ChipItem.
+   *
+   * @param  {function} $$ The createComponent function.
+   * @return {VirtualComponent} The ChipItem component.
+   */
+  render($$) {
+    const {
+      keyword,
+    } = this.props;
+    const textId = keyword.replace(/\s/g, '-');
+    const buttonId = `remove-${textId}`;
+
+    const Container = $$('div');
+
+    // Screen-reader label.
+    const Span = $$('span')
+      .attr('aria-hidden', true)
+      .text(keyword)
+      .setId(textId);
+    Container.append(Span);
+
+    const Button = $$('button')
+      .attr('type', 'button')
+      .attr('aria-label', this.getLabel('Remove'))
+      .attr('aria-labelledby', `${buttonId} ${textId}`)
+      .text('x');
+
+    Button.on('click', () => {
+      this.props.removeKeyword(keyword);
+    });
```

Comment:
Suggestion: For some reason, I read the on-click action being added in another element. So a suggestion to chain it as well.

Does that work?

```suggestion
    const Button = $$('button')
      .attr('type', 'button')
      .attr('aria-label', this.getLabel('Remove'))
      .attr('aria-labelledby', `${buttonId} ${textId}`)
      .text('x')
      .on('click', () => {
        this.props.removeKeyword(keyword);
      });
```